nano/node/transport/udp.hpp

```diff
@@ -69,14 +88,29 @@ namespace transport
 			node_id = node_id_a;
 		}
 
+		nano::transport::transport_type get_type () const
```

Comment:
This converts channel_udp in to more of a channel_supporting_live class, I'd rather see tcp functionality remain in channel_tcp.

I think we should make a new class tcp_channels which holds all live-message tcp channels and is owned by nano::network.

Reply:
Updated with tcp_channels